src/components/BotProfileImage.tsx

```diff
@@ -1,39 +1,43 @@
-import styled from 'styled-components';
+import { styled } from '@linaria/react';
+import { useTheme } from 'styled-components';
 
 import { getColorBasedOnSaturation } from '../colors';
-import Icon from '../icons/bot-profile-image-small.svg';
+import BotProfileIcon from '../icons/bot-profile-image-small.svg';
```

Comment:
Can't this be included in Icon/index.tsx?

Reply:
This is not an icon compatible with UIKit.